### DISCUSSION TITLE
Need new scipy version

### DISCUSSION THREAD

**Discussion by Patcurran** [MechMicroMan/DefDAP#131] [Q&A]
hi, 
So i create a new environment with the lastest version of python (3.12.6). then installed defdap which requires a older scipy version, i know that scipy 1.6.3 works (lastest scipy 1.14.1).  But you cannot install scipy 1.6.3 with python 3.12.6, and you also cannot install older version of python (see [link](https://www.python.org/downloads/)) for security reasons. 

if anyone has any suggestions please let me know.

best wishes,
Patrick

**Comment by JQFonseca** [comment]
Why do you need the latest version of python?

On Sun, 2024-09-22 at 09:19 -0700, Patcurran wrote:
> hi,
> So i create a new environment with the lastest version of python
> (3.12.6). then installed defdap which requires a older scipy version,
> i know that scipy 1.6.3 works (lastest scipy 1.14.1). But you cannot
> install scipy 1.6.3 with python 3.12.6, and you also cannot install
> older version of python (see link [1]) for security reasons.
> if anyone has any suggestions please let me know.
> best wishes,
> Patrick
> —
> Reply to this email directly, view it on GitHub [2], or unsubscribe
> [3].
> You are receiving this because you are subscribed to this
> thread.Message ID: <MechMicroMan/DefDAP/repo-
> ***@***.***>


[1] link https://www.python.org/downloads/
[2] view it on GitHub
    https://github.com/MechMicroMan/DefDAP/discussions/131
[3] unsubscribe

**Reply by Patcurran** [reply]
i dont need the latest version of python, i want less than 3.10 but there are issues with downloading older versions

**Comment by Patcurran** [comment]
i got it sorted found an older version of python
